crates/pop-cli/src/common/builds.rs

```diff
@@ -146,6 +148,43 @@ pub fn guide_user_to_select_profile(cli: &mut impl Cli) -> anyhow::Result<Profil
 	Ok(*prompt.interact()?)
 }
 
+#[cfg(feature = "contract")]
+pub(crate) struct PopComposeBuildArgs;
+#[cfg(feature = "contract")]
+impl ComposeBuildArgs for PopComposeBuildArgs {
+	fn compose_build_args() -> anyhow::Result<Vec<String>> {
+		let mut args: Vec<String> = Vec::new();
+		// match pop related args in pop build --verifiable or pop verify that shouldn't be passed
+		// to Docker image. --path-pos and its following value should be ignored anyway
```

Comment:
Nit: Starting sentences with capital letter, making commands clear with quote (I see this done in some parts here so not super consistent) and I am also a fan of ending sentences with `.` :)

Also simplifying the sentence with removing `and its value`
```suggestion
		// Match pop related args in `pop build --verifiable` or `pop verify` that shouldn't be passed
		// to Docker image. `--path-pos` should be ignored anyway.
```

Could you please fix the rest of the comments below with above feedback too :):)